pack/fhv/fhvc.json

```diff
@@ -570,4 +570,58 @@
     "traits": "Item. Tome.",
     "type_code": "asset"
   }
-]
\ No newline at end of file
+  {
+    "code": "10738",
+    "encounter_code": "myconids",
+    "encounter_position": 1,
+    "enemy_damage": 0,
```

Comment:
```suggestion
```

Reply:
Where there's a 0 value, it's ok to delete the line (to match the card not have a damage icon).